src/asmcnc/comms/router_machine.py

```diff
@@ -2839,11 +2847,34 @@ def set_motor_current(self, axis, current):
                 time.sleep(0.5)
 
             self.send_command_to_motor("STORE TMC PARAMS IN EEPROM", command = STORE_TMC_PARAMS)
-            time.sleep(0.5)
+            time.sleep(1)
             self.tmc_handshake()
-            time.sleep(0.5)
+            time.sleep(1)
             return True
```

Comment:
Is time.sleep() safe? I was always told to use a clock, good conversation either way.

Reply:
I've used it for these functions because I want a blocking delay after commands have been sent (so serial comms thread is not blocked), and don't mind in this instance if it locks up the UI (as we're just doing it on a plain screen at start of Z Head QC). If we were to use it in something customer facing, we might want something that uses clocks (so that UI stays responsive). 

That said, we also don't want any other commands sent while TMC PARAMS are being received, so the block is useful for that reason too. 

So yes, clocks are preferable most of the time, but in this instance I've still chosen to use sleep.

Reply:
yep, I figured that would be the case. Thanks